tests/integration/test_camera_opencv.py

```diff
@@ -17,7 +16,16 @@ def main() -> int:
     """
     Main function.
     """
-    result, device = camera_factory.create_camera(camera_factory.CameraOption.OPENCV, 640, 480)
+    config = camera_config.OpenCVCameraConfig()
+    try:
+        assert config is not None
+    except AssertionError as e:
+        print(f"Configuration test failed: {e}")
+        return -1
```

Comment:
It's fine to just keep the assert line. We don't need to catch exception here, as this is an integration test.